tasks/10-gitops-services.yaml

```diff
@@ -0,0 +1,225 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
This task looks like it has duplicate code from the existing gitops task, what's the difference?